switchbot_ros/msg/CommandTopic.msg

```diff
@@ -0,0 +1,4 @@
+string nickname
```

Comment:
where can we find the definition of switch bot commands?
if they have fixed number of commands, use
```
uint8 ON = 1
uint8 OFF = 2
```
like https://github.com/ros/common_msgs/blob/20a833b56f9d7fd39655b8491a2ec1226d2639b3/actionlib_msgs/msg/GoalStatus.msg#L3